Content.Server/Chat/Managers/ChatManager.cs

```diff
@@ -240,6 +240,35 @@ public void EntityMe(EntityUid source, string action)
             _netManager.ServerSendToMany(msg, clients);
         }
 
+        public void EntityLOOC(EntityUid source, string message)
+        {
+            // Check if entity is a player
+            if (!_entManager.TryGetComponent(source, out ActorComponent? actor))
+            {
+                return;
+            }
+
+            // Check if message exceeds the character limit
+            if (message.Length > MaxMessageLength)
+            {
+                DispatchServerMessage(actor.PlayerSession, Loc.GetString("chat-manager-max-message-length-exceeded-message", ("limit", MaxMessageLength)));
+                return;
+            }
```

Comment:
This is silly, just combine it with the actor check like EntitySay

Reply:
Check is OOC enabled need PlayserSession that exist in actor and now we:
- get actor
- check OOC enabled
- check message length
so it seems there is no need to combine actor getting and message length check

Reply:
Whisper PR is going to fix this anyway so doesn't matter.